src/org/labkey/targetedms/outliers/OutlierGenerator.java

```diff
@@ -247,19 +257,20 @@ private String queryContainerSampleFileRawData(List<QCMetricConfiguration> confi
     public String getMetricLabel(Map<Integer, QCMetricConfiguration> metrics, RawMetricDataSet dataRow)
     {
         QCMetricConfiguration metric = metrics.get(dataRow.getMetricId());
-        String metricLabel;
-        switch (dataRow.getMetricSeriesIndex())
+
+        if (metric.getTraceName() != null)
+        {
+            return metric.getName();
```

Comment:
Can we store the value we want in the series1Label column so we don't need to special case the code here?

Reply:
I am storing YAxisLabel in series1Label column.